src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+interface State {
+  today: string;
+}
+
+export class Clock extends React.Component<Props, State> {
+  state: Readonly<State> = {
+    today: new Date().toUTCString().slice(-12, -4),
```

Comment:
DRY
you do this twice, let's move this to the separate function

Reply:
Moved :white_check_mark: